scripts/kube-list-labels-annotations.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
```

Comment:
I generally try to use `env` which is super common and will use the shell or fallback. 

```suggestion
#!/usr/bin/env bash
```

Reply:
After reviewing this, I don't see anything that prevents you from using `sh` instead of `bash` (ie. associative arrays) I still recommend using `env` to for shell selection.

Reply:
Switched to `/bin/sh` since I removed the bash dependency... I could do `/usr/bin/env sh` I guess, but that feels unnecessary...